{"Id":69,"Message":"{\"Id\":0,\"Message\":\"mesaj\",\"EventId\":1,\"LogLevel\":\"Info\",\"IpAdress\":\"1\",\"HostName\":\"test\",\"CreatedTime\":\"2018-01-12T15:58:42.744623+03:00\"}","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T12:58:42.8179939Z"},
{"Id":70,"Message":"{\"Id\":0,\"Message\":\"mesaj\",\"EventId\":1,\"LogLevel\":\"Info\",\"IpAdress\":\"1\",\"HostName\":\"test\",\"CreatedTime\":\"2018-01-12T16:00:31.4257528+03:00\"}","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T13:00:31.4971362Z"},
{"Id":71,"Message":"{\"Id\":0,\"Message\":\"mesaj\",\"EventId\":1,\"LogLevel\":\"Info\",\"IpAdress\":\"1\",\"HostName\":\"test\",\"CreatedTime\":\"2018-01-12T16:06:10.3236783+03:00\"}","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T13:06:10.3901305Z"},
{"Id":72,"Message":"{\"Id\":0,\"Message\":\"mesaj\",\"EventId\":1,\"LogLevel\":\"Info\",\"IpAdress\":\"1\",\"HostName\":\"test\",\"CreatedTime\":\"2018-01-12T16:06:27.8265691+03:00\"}","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T13:06:27.8273912Z"},
{"Id":73,"Message":"An unhandled exception has occurred while executing the request\nSystem.InvalidCastException: Unable to cast object of type 'Microsoft.AspNetCore.Mvc.PartialViewResult' to type 'Microsoft.AspNetCore.Mvc.ObjectResult'.\r\n   at SampleCoreArch.Web.Filters.FormatReponseFilterAttribute.OnActionExecuted(ActionExecutedContext filterContext) in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\Filters\\FormatReponseFilter.cs:line 30\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T13:06:49.8977276Z"},
{"Id":74,"Message":"An unhandled exception has occurred while executing the request\nSystem.InvalidCastException: Unable to cast object of type 'Microsoft.AspNetCore.Mvc.PartialViewResult' to type 'Microsoft.AspNetCore.Mvc.ObjectResult'.\r\n   at SampleCoreArch.Web.Filters.FormatReponseFilterAttribute.OnActionExecuted(ActionExecutedContext filterContext) in D:\\Store\\VisualStudio\\2017\\Project\\SampleCoreArch\\SampleCoreArch.Web\\Filters\\FormatReponseFilter.cs:line 30\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeNextActionFilterAsync>d__10.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Rethrow(ActionExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ControllerActionInvoker.<InvokeInnerFilterAsync>d__14.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeNextResourceFilter>d__22.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Rethrow(ResourceExecutedContext context)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeFilterPipelineAsync>d__17.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Mvc.Internal.ResourceInvoker.<InvokeAsync>d__15.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Builder.RouterMiddleware.<Invoke>d__4.MoveNext()\r\n--- End of stack trace from previous location where exception was thrown ---\r\n   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()\r\n   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   at Microsoft.AspNetCore.Diagnostics.DeveloperExceptionPageMiddleware.<Invoke>d__7.MoveNext()","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T13:07:21.8079304Z"},
{"Id":75,"Message":"{\"Id\":0,\"Message\":\"mesaj\",\"EventId\":1,\"LogLevel\":\"Info\",\"IpAdress\":\"1\",\"HostName\":\"test\",\"CreatedTime\":\"2018-01-12T16:08:31.8739551+03:00\"}","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T13:08:31.9530809Z"},
{"Id":76,"Message":"{\"Id\":0,\"Message\":\"mesaj\",\"EventId\":1,\"LogLevel\":\"Info\",\"IpAdress\":\"1\",\"HostName\":\"test\",\"CreatedTime\":\"2018-01-12T16:09:38.4594844+03:00\"}","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T13:09:38.5355303Z"},
{"Id":77,"Message":"{\"Id\":0,\"Message\":\"mesaj\",\"EventId\":1,\"LogLevel\":\"Info\",\"IpAdress\":\"1\",\"HostName\":\"test\",\"CreatedTime\":\"2018-01-12T16:18:01.6515231+03:00\"}","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T13:18:01.7302827Z"},
{"Id":78,"Message":"{\"Id\":0,\"Message\":\"mesaj\",\"EventId\":1,\"LogLevel\":\"Info\",\"IpAdress\":\"1\",\"HostName\":\"test\",\"CreatedTime\":\"2018-01-12T16:19:14.4073521+03:00\"}","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T13:19:14.4083014Z"},
{"Id":79,"Message":"{\"Id\":0,\"Message\":\"mesaj\",\"EventId\":1,\"LogLevel\":\"Info\",\"IpAdress\":\"1\",\"HostName\":\"test\",\"CreatedTime\":\"2018-01-12T16:19:24.6364583+03:00\"}","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T13:19:24.6372073Z"},
{"Id":80,"Message":"{\"Id\":0,\"Message\":\"mesaj\",\"EventId\":1,\"LogLevel\":\"Info\",\"IpAdress\":\"1\",\"HostName\":\"test\",\"CreatedTime\":\"2018-01-12T17:07:08.8775213+03:00\"}","EventId":0,"LogLevel":"Error","IpAdress":"192.168.1.101","HostName":"DESKTOP-F285UFK","CreatedTime":"2018-01-12T14:07:08.9491074Z"},
